backend/src/user/user.route.ts

```diff
@@ -0,0 +1,21 @@
+import express from 'express'
+import { celebrate, Joi, Segments } from 'celebrate'
+import { getAllCodes } from './user.service'
+
+const router = express()
+
+router.get(
+	'/:uid/codes',
```

Comment:
Would it be better to place this under something like `/user/:uid/codes`? The [Google documentation](https://developers.google.com/identity/openid-connect/openid-connect) says that the id is `Maximum length of 255 case-sensitive ASCII characters.`, which means this might conflict with other urls we may add in the future.

Right now the id is just a string of numbers, but I think it's good to guard against future changes.